frontend/src/store/modules/SearchStore.ts

```diff
@@ -8,6 +8,7 @@ import GrundschuleModel from "@/types/model/infrastruktureinrichtung/Grundschule
 import MittelschuleModel from "@/types/model/infrastruktureinrichtung/MittelschuleModel";
 import { ActionContext } from "vuex/types/index";
 import { RootState } from "..";
+import { Userinfo } from "@/types/common/Userinfo";
```

Comment:
Der Searchstore hat eine andere fachliche Funktion.

Besser ist es einen eigenen Store für die Userinformationen anzulegen.

In Zukunft werden im UserInfoStore sicher mehr Daten hineinkommen.